[2024-06-23T23:41:41.877+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-23T23:41:41.897+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-23T23:41:34.508993+00:00 [queued]>
[2024-06-23T23:41:41.903+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-23T23:41:34.508993+00:00 [queued]>
[2024-06-23T23:41:41.904+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-23T23:41:41.915+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_nasa_data> on 2024-06-23 23:41:34.508993+00:00
[2024-06-23T23:41:41.922+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1353) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-23T23:41:41.923+0000] {standard_task_runner.py:63} INFO - Started process 1355 to run task
[2024-06-23T23:41:41.923+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_to_s3', 'transform_nasa_data', 'manual__2024-06-23T23:41:34.508993+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/extractAPIdata.py', '--cfg-path', '/tmp/tmp4jdg27yh']
[2024-06-23T23:41:41.925+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask transform_nasa_data
[2024-06-23T23:41:41.965+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-23T23:41:34.508993+00:00 [running]> on host e0fa33b4439a
[2024-06-23T23:41:42.038+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_to_s3' AIRFLOW_CTX_TASK_ID='transform_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T23:41:34.508993+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-23T23:41:34.508993+00:00'
[2024-06-23T23:41:42.039+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-23T23:41:42.059+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-23T23:41:42.060+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/extractAPIdata.py", line 19, in transform_data
    df = pd.DataFrame(response["name"])
                      ~~~~~~~~^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
[2024-06-23T23:41:42.071+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=nasa_to_s3, task_id=transform_nasa_data, run_id=manual__2024-06-23T23:41:34.508993+00:00, execution_date=20240623T234134, start_date=20240623T234141, end_date=20240623T234142
[2024-06-23T23:41:42.083+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 20 for task transform_nasa_data ('NoneType' object is not subscriptable; 1355)
[2024-06-23T23:41:42.097+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-23T23:41:42.119+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-23T23:41:42.121+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
